src/main/java/com/cincialert/controllers/CinciAlertController.java

```diff
@@ -0,0 +1,24 @@
+package com.cincialert.controllers;
+import com.cincialert.service.IUserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class CinciAlertController {
+    private final IUserService userService;
+
+    @Autowired
+    public CinciAlertController(IUserService userService) {
+        this.userService = userService;
+    }
+
+    @RequestMapping("/")
```

Comment:
Fixed the issue of project displaying an error on startup and navigation to "localhost:8080"